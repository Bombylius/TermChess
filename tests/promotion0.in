a2 a4 c7 c5 a4 a5 c5 c4 a5 a6 c4 c3 a6 b7 c3 b2 b7 a8 x b7 a8 r b2 c1 n .q 
