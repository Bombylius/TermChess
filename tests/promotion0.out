1103806595072
67371008
16777216
17179869184
65536
4398046511104
512
2814749767106560
5
5
360287970189639680
Rnbqkbnr/p2ppppp/8/8/8/8/2PPPPPP/RNnQKBNR w KQk - 0 6
